Src/FluentAssertions/Events/EventMonitor.cs

```diff
@@ -45,23 +45,18 @@ public OccurredEvent[] OccurredEvents
                 IEnumerable<OccurredEvent> query =
                     from mapItem in recorderMap.ToArray()
```

Comment:
Since this no longer uses the `Value` we can use `recorderMap.Keys` instead.

`public EventMetadata[] MonitoredEvents` can also use `.Values` instead of `.ToArray()` as it does not use `Key`.